src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -911,9 +911,19 @@ static void applyRepairCommandPoolSize(Config config)
     }
 
     @VisibleForTesting
-    static void applyTrackWarningsValidations(Config config)
+    static void applyReadThresholdsValidations(Config config)
     {
-        config.track_warnings.validate("track_warnings");
+        validateReadThresholds("coordinator_read_size", config.coordinator_read_size_warn_threshold, config.coordinator_read_size_fail_threshold);
+        validateReadThresholds("local_read_size", config.local_read_size_warn_threshold, config.local_read_size_fail_threshold);
+        validateReadThresholds("row_index_read_size", config.row_index_read_size_warn_threshold, config.row_index_read_size_fail_threshold);
+    }
+
+    private static void validateReadThresholds(String name, DataStorageSpec warn, DataStorageSpec fail)
```

Comment:
Are fine with accepting 0 as a valid value, even if it makes the db unusable? Probably any other very low value is going to be as bad as 0, so I guess that rejecting it wouldn't make a big difference.

Reply:
yeah, 0b and 1b are basically the same, so don't think blocking 0 matters much.

2 checks are memory based (coordinator and index), so those also depend on the JVM running, so not sure if a lower bound would make any sense